src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/definition/CommonCaseFields.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition;
+
+/**
+ * Names of the fields used by Bulk Scan in service CCD case definitions.
+ */
+public class CommonCaseFields {
+
+    private CommonCaseFields() {
+        // hiding the constructor - this class is for constants only
```

Comment:
just utility class construct :wink:. and class is `final` then

Reply:
Will change

Reply:
done - class has been removed